lib/main.dart

```diff
@@ -7,6 +8,10 @@ import 'package:skitour_planning/piechart/piechart_home_screen.dart';
 
 import 'examples/examples_screen.dart';
 
+var logger = Logger(
+  printer: PrettyPrinter(),
+);
+
```

Comment:
If you're not using anywhere here don't add it. Also, nothing should be initiated before the main function.